src/java/picard/analysis/AlignmentSummaryMetricsCollector.java

```diff
@@ -318,11 +323,20 @@ else if(doRefMetrics) {
 
                                 // With both reads mapped we can see if this pair is chimeric
                                 if (Math.abs(record.getInferredInsertSize()) > maxInsertSize ||
-                                        !record.getReferenceIndex().equals(record.getMateReferenceIndex())) {
+                                        !record.getReferenceIndex().equals(record.getMateReferenceIndex()) ||
+                                        SamPairUtil.getPairOrientation(record) != expectedOrientation ||
```

Comment:
I must admit that I don't know how exactly we define "chimeric" reads, but I just want to confirm with everyone:
this will count read pairs that overlap structural variants as chimeric.  E.g.  an inversion will cause reads to have unexpected orientations and now we're saying that those are chimeric.  (Although I suppose we might have already been doing that depending on the value for max insert size, as that could be capturing large deletions)

I would like @tfenne and @yfarjoun to confirm please.